src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -950,7 +958,7 @@ internal string GetCookieHeader(Uri uri, out string optCookie2)
 
             string delimiter = string.Empty;
 
-            var builder = StringBuilderCache.Acquire();
+            StringBuilder builder = StringBuilderCache.Acquire();
```

Comment:
FYI for ask-mode changes, we try to avoid unnecessary unrelated changes like this. Reasons are (1) distracting from actual diff (2) in general, adds risk. Not worth changing this now, especially because of the volume of other changes...